TypeCobol/Compiler/Sql/Model/SelectClause.cs

```diff
@@ -39,9 +40,15 @@ public class UnpackedRowSelection : Selection
         public override SelectionType Type => SelectionType.UnpackedRow;
     }
 
-    public class DotStarSelection : Selection
+    public class TableOrViewAllColumnsSelection : Selection
     {
-        public override SelectionType Type => SelectionType.DotStar;
+        public TableViewCorrelationName TableOrViewOrCorrelationName { get; }
```

Comment:
If `TableViewCorrelationName` is an SqlObject, Selection and all its derived classes must inherit from SqlObject too. Otherwise the visitor pattern won't work